src/lib.rs

```diff
@@ -8,39 +8,50 @@
 //! Minimal example:
 //!
 //! ```rust
+//! #[macro_use]
 //! extern crate yew;
-//! use yew::html::*;
-//!
+//! use yew::prelude::*;
+//! 
 //! struct Model {
 //!     value: i64,
 //! }
-//!
+//! 
 //! enum Msg {
 //!     DoIt,
 //! }
-//!
-//! fn update(context: &mut Context<Msg>, model: &mut Model, msg: Msg) {
-//!     match msg {
-//!         Msg::DoIt => {
-//!             model.value = model.value + 1;
+//! 
+//! impl Component for Model {
+//!     type Message = Msg;
+//!     type Properties = ();
```

Comment:
Would it be better adding another example that is showing how to use `Properties`?

Reply:
Such an example requires at least two component, doesn't it? It's a big one, I don't think it is suitable in `lib.rs`. We have plenty of such examples in `/examples/*`.

Reply:
@huangjj27 Thank you for the idea. I think it's better to remain `()` for `Properties` to show you could avoid this parameter for application components.